lib/takeaway.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'texter'
+
+class Takeaway
+  attr_reader :menu, :order
+
+  def initialize(order = [], menu = Menu.new, texter = Texter.new)
+    @order = order
+    @menu = menu
+    @texter = texter
```

Comment:
Did you do menu and text like this with default variables to allow the possibility of having another menu passed in?